config/hof-defaults.js

```diff
@@ -36,7 +36,31 @@ const defaults = {
   apis: {
     pdfConverter: process.env.PDF_CONVERTER_URL
   },
-  serveStatic: process.env.SERVE_STATIC_FILES !== 'false'
+  serveStatic: process.env.SERVE_STATIC_FILES !== 'false',
+  sanitisationBlacklistArray: {
+    // Input will be sanitised using the below rules
+    // The key is what we're sanitising out
+    // The regex is the rule we used to find them (note some dictate repeating characters)
+    // And the replace is what we're replacing that pattern with. Usually nothing sometimes a
+    // single character or sometimes a single character followed by a "-"
+    '/*': { regex: '\/\\*', replace: '' },
+    '*/': { regex: '\\*\\/', replace: '' },
+    '|': { regex: '\\|', replace: '' },
+    '&&': { regex: '&&+', replace: '&' },
+    '@@': { regex: '@@+', replace: '@' },
+    '/..;/': { regex: '/\\.\\.;/', replace: '' }, // Purposely input before ".." as they conflict
+    '..': { regex: '\\.\\.+', replace: '.' },
+    '/etc/passwd': { regex: '\/etc\/passwd', replace: '' },
+    'c:\\': { regex: 'c:\\\\', replace: '' },
+    'cmd.exe': { regex: 'cmd\\.exe', replace: '' },
+    '<': { regex: '<', replace: '<-' },
+    '>': { regex: '>', replace: '>-' },
+    '[': { regex: '\\[+', replace: '[-' },
+    ']': { regex: '\\]+', replace: ']-' },
+    '~': { regex: '~', replace: '~-' },
+    '&#': { regex: '&#', replace: '&#-' },
+    '%U': { regex: '%U', replace: '%U-' }
+  }
```

Comment:
naxsi stuff is private, should we do the same with this?

Reply:
Pfft, I'd say possibly not. Mainly because it would be a pain to test locally etc. I think this should be fine as naxsi bans traffic where in this case we are just escaping certain things. It might be worth putting this into its own config file though.

Reply:
Good point 👍

Reply:
Thanks chaps, will move into its own config as suggested.